pom.xml

```diff
@@ -107,6 +150,11 @@
             <artifactId>bigqueue</artifactId>
             <version>1.7.0.4</version>
         </dependency>
+        <dependency>
```

Comment:
I tried.. it is not shaded because its "provided" on bigqueue.. so i packed that as well.